lib/vsc/install/shared_setup.py

```diff
@@ -1513,6 +1518,28 @@ def parse_target(self, target, urltemplate=None):
                     new_target['dependency_links'] += [''.join([git_scheme, url, '/hpcugent/', dep_name, '.git#egg=',
                                                                 dep_name_version])]
 
+        if 'VSC_RPM_PYTHON' in os.environ:
+            for key, snrs in vsc_filter_rpm.items():
+                def snr(txt):
```

Comment:
What is `snr` short for? Is it "search 'n replace"?
Oh my, please rename to something more obvious, like `search_replace` or whatever...

Making `snr` a function, and `snrs` a list of things to iterate over with `snr` (rather a list of `snr`s) is ... confusing.